README.md

```diff
@@ -38,13 +38,14 @@ Key features:
 - Deploy MCAD using steps [here](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/main/doc/deploy/deployment.md).
 
 - In Visual Studio Code update `.vscode/launch.json` so that `"KUBECONFIG"` points to your Kubernetes config file.<br>
-- If you changed the namespace in `instascale-configmap.yaml` update the `args[]` in `launch.json` to include `--configs-namespace="<YOUR_NAMESPACE>"`.<br>
+- If you changed the namespace in `instascale_configmap.yaml` update the `args[]` in `launch.json` to include `--configs-namespace="<YOUR_NAMESPACE>"`.<br>
```

Comment:
I think the proper way of changing the namespace, when InstaScale is deployed with `make deploy`, is to have it changed globally here:

https://github.com/project-codeflare/instascale/blob/cc2a63c5535de89ed38d43746251bc3c9bc3c9ce/config/default/kustomization.yaml#L2

Reply:
Sorry that's right, great spot Antonin :)

Reply:
Thanks @astefanutti, 
This has been updated

Reply:
Agreed, this can be re-worked later if needed.